charts/cass-operator/templates/deployment.yaml

```diff
@@ -30,11 +30,9 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           volumeMounts:
-            {{- if .Values.admissionWebhooks.enabled }}
```

Comment:
Will we be using this for other admissionWebhook enablement?
`.Values.admissionWebhooks.enabled`

Reply:
Thinking if not, we can remove it from values.yaml, looks like this may be only place we use it.

https://github.com/k8ssandra/k8ssandra/blob/0ff888bcfa27a66be8876f7214f207ed9d4a4a75/charts/cass-operator/values.yaml

Reply:
There are no other webhooks at this time. cass-operator expects the volume mount to exist even if the webhook is disabled.